tests/testthat/test-data_frame-osm.R

```diff
@@ -221,6 +221,34 @@ test_that ("date", {
     expect_null (meta_l$meta_no_call$query_type)
 })
 
+test_that ("out tags center", {
+    # q <- getbb ("Franja de Ponent", featuretype = "relation") %>%
+    bb <- rbind (c (-0.73, 1.27), c (40.63, 42.63))
+    rownames (bb) <- c ("x", "y")
+    colnames (bb) <- c ("min", "max")
+    q <- opq (bb, out = "tags center") %>%
+        add_osm_feature ("amenity", "community_center")
+
+    osm_tags_center <- test_path ("fixtures", "osm-tags_center.osm")
+    doc <- xml2::read_xml (osm_tags_center)
+
+    expect_silent (x <- osmdata_data_frame (opq_string_intern (q), doc))
+    expect_silent ( x_no_call <- osmdata_data_frame (doc = doc))
+
+    cols <- c (
+        "osm_type", "osm_id", "osm_center_lat", "osm_center_lon", "addr:city",
+        "addr:housenumber", "addr:postcode", "addr:street", "amenity",
+        "building", "building:colour", "building:levels", "building:material",
+        "community_centre", "community_centre:for", "name", "name:ca",
+        "old_name", "operator", "roof:material", "roof:shape", "sport", "type",
+        "website", "wikidata", "wikipedia"
+    )
+    expect_named (x, cols)
+    expect_named (x_no_call, cols)
+    expect_s3_class (x, "data.frame")
+    expect_s3_class (x_no_call, "data.frame")
```

Comment:
Ok, thanks! I will skip the last two tests, because in some cases can be false (I think). `center` returns the coordinates of the center of the bounding box of the object, and I think overpass return the object even if some parts fall outside the bounding box of the query.